test/rendering/__fixtures__/scenario_1.fix.js

```diff
@@ -0,0 +1,102 @@
+export default () => ({
```

Comment:
Now the logs on circle say `scenario_1`, should have a more descriptive name.

Reply:
Yes, I agree. when we go through each feature to create a fixture file, we should have a more descriptive name. Not this time though, this fixture file which is randomly created will be removed in the future anyway.

Reply:
ok fine